modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/utils/utils.es.js

```diff
@@ -31,3 +31,14 @@ export const isEqualObjects = (firstObj, secondObj) => {
 export const getValidName = (defaultName, name) => {
 	return name && name.toLowerCase() !== 'null' ? name : defaultName;
 };
+
+export const getLocalizableValues = (localizableValue, defaultName = '') => {
```

Comment:
@kevenleone I think we can rename this function based on his return. Something like normalizeNames. What do you think?

Reply:
Totally agreed :p